src/Components/test/E2ETest/Tests/FormsTest.cs

```diff
@@ -249,6 +249,87 @@ public void InputDateInteractsWithEditContext_NullableDateTimeOffset()
             Browser.Empty(messagesAccessor);
         }
 
+        [Fact]
+        public void InputDateInteractsWithEditContext_TimeInput()
+        {
+            var appElement = MountTypicalValidationComponent();
+            var departureTimeInput = appElement.FindElement(By.ClassName("departure-time")).FindElement(By.TagName("input"));
+            var messagesAccessor = CreateValidationMessagesAccessor(appElement);
+
+            // Validates on edit
+            Browser.Equal("valid", () => departureTimeInput.GetAttribute("class"));
+            departureTimeInput.SendKeys("06:43\t");
+            Browser.Equal("modified valid", () => departureTimeInput.GetAttribute("class"));
+
+            // Can become invalid
+            ApplyInvalidInputDateValue(".departure-time input", "01:234:56");
+            Browser.Equal("modified invalid", () => departureTimeInput.GetAttribute("class"));
+            Browser.Equal(new[] { "The DepartureTime field must be a date." }, messagesAccessor);
```

Comment:
Should the default message be sensitive to the type? Should this say `must be a time`, for example?

Reply:
Oh I see you commented about that elsewhere too.

Reply:
That was a question I had as well. I think it probably should be sensitive to the input type. I'll move forward with this change for now.